cmd/paratime/list.go

```diff
@@ -18,7 +21,7 @@ var listCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg := cliConfig.Global()
 		table := table.New()
-		table.SetHeader([]string{"Network", "Paratime", "ID"})
+		table.SetHeader([]string{"Network", "Paratime", "ID", "DENOMINATION(S)"})
```

Comment:
```suggestion
		table.SetHeader([]string{"Network", "Paratime", "ID", "Denomination(s)"})
```
AFAIK capitalization happens during rendering.